src/Message/AuthorizeRequest.php

```diff
@@ -188,6 +232,18 @@ public function setStatementDescriptor($value)
         return $this->setParameter('statementDescriptor', $value);
     }
 
+    public function getStatementDescriptorSuffix()
+    {
+        return $this->getParameter('statementDescriptorSuffix');
```

Comment:
Keep this `statementDescriptor`. Make the driver do the translation to `statement_descriptor_suffix`.

Reply:
done